crypto/compactcert/builder_test.go

```diff
@@ -17,11 +17,16 @@
 package compactcert
 
 import (
+	"context"
+	"database/sql"
 	"fmt"
+	"github.com/algorand/go-algorand/util/db"
```

Comment:
this needs to go to line 32, with the algorand import block, alphabetically sorted.